styles/_settings.scss

```diff
@@ -38,9 +43,13 @@ $font-weight-bold: 700;
 $font-size-small: 10px;
 $font-size-regular: 12px;
 $font-size-x-regular: 13px;
+$font-size-xx-regular: 16px;
```

Comment:
Yeah.. I am waiting for the UI Kit to discard sizes/colours... But since we don't have it yet I don't want to mess it up :/